aten/src/ATen/functorch/BatchRulesDecompositions.cpp

```diff
@@ -262,8 +262,6 @@ TORCH_LIBRARY_IMPL(aten, FuncTorchBatched, m) {
   OP_DECOMPOSE(t_);
   OP_DECOMPOSE(swapdims_);
   OP_DECOMPOSE(swapaxes_);
-  OP_DECOMPOSE(unfold_copy);
```

Comment:
not a composite implicit autograd op, so shouldn't be here 

(This was caught through a segfault at test_vjpvmap_unfold_copy_cpu_float32)

Reply:
Thanks for catching that